stp_zmq/zstack.py

```diff
@@ -821,7 +821,7 @@ def transmitThroughListener(self, msg, ident) -> Tuple[bool, Optional[str]]:
             return False, err_str
         except Exception as e:
             err_str = '{}{} got error {} while sending through listener to {}' \
-                .format(CONNECTION_PREFIX, self, e, z85_to_friendly(ident))
```

Comment:
I would recommend to add a check (ValueError catching) in `z85_to_friendly`, so that we don't fail in case of non-z85 passed

Reply:
ok